readme.md

```diff
@@ -287,6 +287,47 @@ hardware. If you are facing problems, please ask for support.
 
 - That being said, a few advices to anyone facing Windows problems:
   - if `john` is not recognizing your GPU card (and you are sure all required GPU drivers are installed):
+    - using the `detect-OpenCL.ps1` script contained in the main folder:
+      ```powershell
+      # I downloaded and extracted "john" to c:\Temp\Devel.
```

Comment:
Regarding `c:\Temp\Devel`, this would be unsafe on a multi-user Windows system (and may be mildly unsafe on single-user as well, where pseudo-user accounts may exist to run certain services), so I suggest that we avoid "promoting" such bad practices. For this demo, perhaps you could just stay in `C:\Users\Me` and say that you assume that's where the JtR download was extracted to.